src/App.scss

```diff
@@ -0,0 +1,2 @@
+$pf-global--enable-font-overpass-cdn: true;
```

Comment:
No need to have this file, check the latest master this file is strill not relevant.